bit/test/test_api.lua

```diff
@@ -0,0 +1,55 @@
+local bit = require("bit")
+
+function Test_and(t)
+    local result, err = bit.band(1, 0)
+    assert(not err, err)
```

Comment:
FYI - consider using the testify assertions if you like… they can be a little less boilerplate (they add the default message).

```go
local assert = require 'assert'
-- ...
assert:Equal(t, expected, actual)
```

- https://github.com/vadv/gopher-lua-libs/blob/master/base64/test/test_api.lua#L3
- https://github.com/vadv/gopher-lua-libs/blob/master/base64/test/test_api.lua#L70-L75

Reply:
When comparing numbers, try EqualValues (which doesn't care about int vs uint32, etc)